core/src/main/scala/cromwell/util/JsonFormatting/WdlValueJsonFormatter.scala

```diff
@@ -18,6 +18,8 @@ object WdlValueJsonFormatter extends DefaultJsonProtocol {
       case m: WdlMap => new JsObject(m.value map {case(k,v) => k.valueString -> write(v)})
       case e: WdlExpression => JsString(e.toWdlString)
       case q: WdlPair => new JsObject(Map("left" -> write(q.left), "right" -> write(q.right)))
+      case WdlOptionalValue(_, Some(opt)) => write(opt)
+      case WdlOptionalValue(_, None) => JsString("null")
```

Comment:
@kshakir that's a very good point.  Aesthetically I prefer no k/v, but I bet an explicit k/"null" will probably reduce support questions.  So I'm not sure which to vote for...